src/components/Effects.js

```diff
@@ -14,22 +16,22 @@ export default function Effects({
 
   return (
     <div className="Function-Board">
-      <div className="Amp-bar">
-        <h2>SET AUDIO EFFECTS</h2>
+      <div className="Effects-Bar">
+        <p className="Effects-Headline">SET AUDIO EFFECTS</p>
       </div>{" "}
-      <h2> Reverb </h2>
+      <p className="Effect-Value"> REVERB </p>
```

Comment:
I would recommend to work here a bit semantically. And using a `label` element with the input.